routes/api.php

```diff
@@ -16,8 +18,9 @@
 |
 */
 
+Route::patch('/booking-setting', [SettingsController::class, 'storeBookingGeneralInformation']);
```

Comment:
this route should definitely be inside the auth middleguard

Reply:
ohh true, forgot to change it after the axios fix

Reply:
no worries, smol fix